### ISSUE TITLE
Issues with comment text selection

### ISSUE THREAD

**Issue by kchadha** [LLK/scratch-blocks#1623]
### Expected Behavior

Comment text should not get selected if not interacting with that comment.

### Actual Behavior

Comment text (on all comments) gets selected when resizing a comment on the workspace.

![comment-text-selection](https://user-images.githubusercontent.com/1786240/42338667-966e4be2-8058-11e8-89e7-cee65453e375.gif)

**Comment by rachel-fenichel** [comment]
Does this also happen upstream?

**Comment by kchadha** [comment]
@rachel-fenichel, yes it does. I tested on the [blockly playground](https://blockly-demo.appspot.com/static/tests/playground.html)

**Comment by rachel-fenichel** [comment]
Probably needs something like this, or need to update this in css.js to include comment text:

```
  /*
    Don't allow users to select text.  It gets annoying when trying to
    drag a block and selected text moves instead.
  */
  '.blocklySvg text, .blocklyBlockDragSurface text {',
    'user-select: none;',
    '-moz-user-select: none;',
    '-ms-user-select: none;',
    '-webkit-user-select: none;',
    'cursor: inherit;',
  '}',
```
